components/application-connectivity-validator/internal/validationproxy/handler.go

```diff
@@ -295,7 +295,8 @@ func createReverseProxy(destinationHost string, reqOpts ...requestOption) *httpu
 		Director: func(request *http.Request) {
 			request.URL.Scheme = "http"
 			request.URL.Host = destinationHost
-
+			request.Host = ""
```

Comment:
Are you certain we can safely alter the Host header for _every_ request?

Reply:
the host header will be updated to the correct value automatically, but this only works if it is not specified before

Reply:
Yes I get that, I made that change originally, but the application validator serves more than just the eventing endpoints. Hence my question: can we safely replace the original header ("...xip.io") with the internal host name? Most likely yes, but it's worth asking if we really checked every scenario.

Reply:
Right, it works for all the other scenarios.